modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -153,12 +153,16 @@ Console components include API Console, API Console Documentation, API Console I
 
 The component shows the API instances and documentation from xref:exchange::index.adoc[Anypoint Exchange].
 
+When users need a client ID and client secret to access an API, they can select an API instance in the *Credential source* list to populate the client ID and client secret from that instance automatically, or select *Custom credentials* in the list to enter the client ID and client secret manually.
```

Comment:
Instead of 'API Instance' we should use `client application`

Reply:
Thank you! Fixed.